pandas/_libs/tslibs/timezones.pyx

```diff
@@ -45,7 +38,7 @@ from pandas._libs.tslibs.util cimport (
 
 cdef int64_t NPY_NAT = get_nat()
 cdef tzinfo utc_stdlib = timezone.utc
-cdef tzinfo utc_pytz = pytz.utc
+cdef object utc_pytz = pytz.UTC if pytz else None
```

Comment:
cant we keep this as a tzinfo object?

Reply:
Ah I had thought cython wouldn't like tzinfo since this could be `None`, but was able to keep as tzinfo